src/component/Inputcity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import Time from './Time'
+import Weatherrr from './Weatherrr'
+const weather = require('openweather-apis');
+
+function Inputcity() {
+    const [cooord, setcooord] = useState({ lati: '51.5085', long: '-0.1257', cityy: 'London' })
+    const [city, setcity] = useState('London')
+    const submit = () => {
+        weather.setLang('en');
+        weather.setCity(`${city}`);
+        weather.setUnits('metric');
+        weather.setAPPID('f7d454f5834522dcb92bcf5d38a1db14');
```

Comment:
there is something called env variables, try to use them

the library called dotenv

Reply:
```suggestion
        weather.setAPPID(process.env.APPID);
```